module/Activity/src/Controller/AdminCategoryController.php

```diff
@@ -92,7 +92,7 @@ public function deleteAction()
         $request = $this->getRequest();
 
         if ($request->isPost()) {
-            $categoryId = (int) $this->params('id');
+            $categoryId = (int)$this->params('id');
```

Comment:
I think we have been changing these type casts many times. We should really decide what to do; space, no space, or no type cast at all (if the route is correctly specified).

Reply:
Fixed this and added it to Phpstorm code style